media-video/fuppes/fuppes-1.0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Free UPnP A/V media server"
+DESCRIPTION="FUPPES is a free, multiplatform UPnP A/V Media Server. \
+FUPPES supports a wide range of UPnP MediaRenderers as well as on-the-fly \
+transcoding of various audio, video and image formats. FUPPES also includes \
+basic DLNA support."
+HOMEPAGE="https://github.com/uvoelkel/fuppes/"
+COPYRIGHT="1989-1991 Free Software Foundation"
```

Comment:
2007-2009 Ulrich Völkel
remove the FSF one as that is likely just from one of the build tools
I normal run grep on the source directory searching for "copyright" and then investigate the files that come up in that list.
`
grep -nri "copyright" 
`